config/endpoints/api_particulier/cnous_student_scholarship.yml

```
@@ -8,20 +8,22 @@
     entity_type_description: |+
       Cette API délivre **uniquement les bourses "obligatoires"** et ne concerne pas les bourses d'ordre facultatif.
 
-      L'API délivre les données :
-      - ✅ des **boursiers d’État sur critères sociaux (gérés par les Crous)** ;
-      - ✅ des **boursiers sur critères sociaux des filières sanitaires et sociales** des régions Normandie et Occitanie. D'autres régions devraient être couvertes à l'avenir.
+      L'API délivre les données sur :
+      - ✅ **le statut de bourse définitif pour les boursiers d’État sur critères sociaux (gérés par les Crous)** ;
+      - ✅ **le statut de bourse provisoire et définitif pour les boursiers sur critères sociaux des filières sanitaires et sociales** des régions Occitanie, Grand Est, Bourgogne-Franche-Comté, Bretagne et Auvergne-Rhône-Alpes. D'autres régions devraient être couvertes à l'avenir.
 
     geographical_scope_description:  |+
       L'API couvre :
-      - ✅ les territoire français, dont les ✅ DROM ; _uniquement régions Normandie et Occitanie pour les bourses des filières sanitaires et sociales_ ;
-      - ✅ les boursiers de nationalité française.
+      - ✅ la France Hexagonale et la Corse pour les bourses d'Etat; 
+      - ✅ les régions Occitanie, Grand Est, Bourgogne-Franche-Comté, Bretagne et Auvergne-Rhône-Alpes pour les bourses des filières sanitaires et sociale.
 
       Ne sont pas couverts par cette API :
       - ❌ les territoires d'outre-mer ;
-      - ❌ les données des étudiants boursiers étrangers.
+      - ❌ les données des étudiants ayant des bourses étrangères.
     updating_rules_description: |+
-      Les données sont **mises à jour une fois par an**, début septembre, une fois tous les étudiants inscrits.
+      Les données sont **mises à jour tout au long de l'année**, avec une bascule d'année qui se fait à partir du mois de : 
```

@Miryad3108 qu'est-ce qu'on entend par bascule ?

Est-ce que le paragraphe suivant te semble bon ?

```suggestion
      Les données sont **mises à jour tout au long de l'année**. Une bascule d'année s'effectue à la fin de l'année scolaire, les données concernant la rentrée suivante sont alors disponibles (et remplacent l'année en cours) à partir du mois de : 
```

---

par bascule on entend un changement d'année (année scolaire 2023-2024 à 2024-2025 par exemple) 
@DorineLam